### ISSUE TITLE
Ticket requester automatically assigned

### ISSUE THREAD

**Issue by bielu22** [glpi-project/glpi#11070]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

GLPI 10.0.0-rc2

### Bug description

When I create a new ticket, I am always automatically added as a requester in addition to the user I choose. In the "Requester" field, only the user I chose is visible, my username appears only after I have saved the ticket.

### Relevant log output

```shell
Couldn't find any
```

### Page URL

https://localhost/glpi/front/preference.php

### Steps To reproduce

My settings
Pre-Select me as Requester when creating ticket set to No
General settings
Pre-Select me as Requester when creating ticket set to No
Create ticket
Save ticket
I am automatically added as a requester

### Your GLPI setup information

<details><summary>Information about system installation and configuration</summary><pre>GLPI 10.0.0-rc2 ( => /home/www/glpi)
Installation mode: TARBALL
Current language:en_GB

</details><details><summary>Server</summary><pre> 
Operating system: Linux xxxx.xxxx.xxxx.xx 5.16.15-201.fc35.x86_64 #1 SMP PREEMPT Thu Mar 17 05:45:13 UTC 2022 x86_64
PHP 8.0.16 fpm-fcgi (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, bz2, calendar, cgi-fcgi, ctype, curl, date, dom,
	exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, intl, json, ldap, libxml, mbstring, mysqli, mysqlnd, openssl, pcre,
	pdo_mysql, pdo_sqlite, posix, session, shmop, sockets, sodium, sqlite3, standard, sysvmsg, sysvsem, sysvshm, tokenizer, xml,
	xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib)
Setup: max_execution_time="600" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="32M" 
Software: Apache/2.4.52 (Fedora Linux) OpenSSL/1.1.1l ()
	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.74 Safari/537.36
Server Software: MariaDB Server
	Server Version: 10.5.13-MariaDB
	Server SQL Mode: STRICT_TRANS_TABLES,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
	Parameters: root@localhost/glpi
	Host info: Localhost via UNIX socket
	
PHP version (8.0.16) is supported.
Sessions configuration is OK.
Allocated memory is sufficient.
mysqli extension is installed.
Following extensions are installed: dom, fileinfo, json, simplexml.
curl extension is installed.
gd extension is installed.
intl extension is installed.
libxml extension is installed.
zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (10.5.13) is supported.
The log file has been created successfully.
Write access to /home/www/glpi/files/_cache has been validated.
Write access to /home/www/glpi/config has been validated.
Write access to /home/www/glpi/files/_cron has been validated.
Write access to /home/www/glpi/files has been validated.
Write access to /home/www/glpi/files/_dumps has been validated.
Write access to /home/www/glpi/files/_graphs has been validated.
Write access to /home/www/glpi/files/_lock has been validated.
Write access to /home/www/glpi/files/_pictures has been validated.
Write access to /home/www/glpi/files/_plugins has been validated.
Write access to /home/www/glpi/files/_rss has been validated.
Write access to /home/www/glpi/files/_sessions has been validated.
Write access to /home/www/glpi/files/_tmp has been validated.
Write access to /home/www/glpi/files/_uploads has been validated.
Web access to the files directory should not be allowed but this cannot be checked automatically on this instance.
Make sure access to error log file (/files/_log/php-errors.log) is forbidden; otherwise review .htaccess file and web server configuration.
For security reasons, SELinux mode should be Enforcing.
exif extension is installed.
ldap extension is installed.
openssl extension is installed.
zip extension is installed.
bz2 extension is installed.
Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to /home/www/glpi/marketplace has been validated.
Timezones seems loaded in database.

</details><details><summary>GLPI constants</summary><pre> 
GLPI_ROOT: /home/www/glpi
GLPI_CONFIG_DIR: /home/www/glpi/config
GLPI_VAR_DIR: /home/www/glpi/files
GLPI_MARKETPLACE_DIR: /home/www/glpi/marketplace
GLPI_USE_CSRF_CHECK: 1
GLPI_CSRF_EXPIRES: 7200
GLPI_CSRF_MAX_TOKENS: 100
GLPI_USE_IDOR_CHECK: 1
GLPI_IDOR_EXPIRES: 7200
GLPI_ALLOW_IFRAME_IN_RICH_TEXT: 
GLPI_TELEMETRY_URI: https://telemetry.glpi-project.org
GLPI_INSTALL_MODE: TARBALL
GLPI_NETWORK_MAIL: glpi@teclib.com
GLPI_NETWORK_SERVICES: https://services.glpi-network.com
GLPI_MARKETPLACE_ALLOW_OVERRIDE: 1
GLPI_MARKETPLACE_MANUAL_DOWNLOADS: 1
GLPI_USER_AGENT_EXTRA_COMMENTS: 
GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE: 1
GLPI_AJAX_DASHBOARD: 1
GLPI_CALDAV_IMPORT_STATE: 0
GLPI_DEMO_MODE: 0
GLPI_CENTRAL_WARNINGS: 1
GLPI_DOC_DIR: /home/www/glpi/files
GLPI_CACHE_DIR: /home/www/glpi/files/_cache
GLPI_CRON_DIR: /home/www/glpi/files/_cron
GLPI_DUMP_DIR: /home/www/glpi/files/_dumps
GLPI_GRAPH_DIR: /home/www/glpi/files/_graphs
GLPI_LOCAL_I18N_DIR: /home/www/glpi/files/_locales
GLPI_LOCK_DIR: /home/www/glpi/files/_lock
GLPI_LOG_DIR: /home/www/glpi/files/_log
GLPI_PICTURE_DIR: /home/www/glpi/files/_pictures
GLPI_PLUGIN_DOC_DIR: /home/www/glpi/files/_plugins
GLPI_RSS_DIR: /home/www/glpi/files/_rss
GLPI_SESSION_DIR: /home/www/glpi/files/_sessions
GLPI_TMP_DIR: /home/www/glpi/files/_tmp
GLPI_UPLOAD_DIR: /home/www/glpi/files/_uploads
GLPI_INVENTORY_DIR: /home/www/glpi/files/_inventories
GLPI_NETWORK_REGISTRATION_API_URL: https://services.glpi-network.com/api/registration/
GLPI_MARKETPLACE_PLUGINS_API_URI: https://services.glpi-network.com/api/glpi-plugins/
GLPI_I18N_DIR: /home/www/glpi/locales
GLPI_VERSION: 10.0.0-rc2
GLPI_SCHEMA_VERSION: 10.0.0-rc2@26641609f9b8fcb8038fb220c3fcafa75a95021f
GLPI_MARKETPLACE_PRERELEASES: 1
GLPI_MIN_PHP: 7.4.0
GLPI_MAX_PHP: 8.2.0
GLPI_YEAR: 2022

</details><details><summary>Libraries</summary><pre> 
htmlawed/htmlawed version 1.2.6 in (/home/www/glpi/vendor/htmlawed/htmlawed)
phpmailer/phpmailer version 6.6.0 in (/home/www/glpi/vendor/phpmailer/phpmailer/src)
simplepie/simplepie version 1.5.8 in (/home/www/glpi/vendor/simplepie/simplepie/library)
mpdf/mpdf in (/home/www/glpi/vendor/mpdf/mpdf/src)
michelf/php-markdown in (/home/www/glpi/vendor/michelf/php-markdown/Michelf)
true/punycode in (/home/www/glpi/vendor/true/punycode/src)
iamcal/lib_autolink in (/home/www/glpi/vendor/iamcal/lib_autolink)
sabre/dav in (/home/www/glpi/vendor/sabre/dav/lib/DAV)
sabre/http in (/home/www/glpi/vendor/sabre/http/lib)
sabre/uri in (/home/www/glpi/vendor/sabre/uri/lib)
sabre/vobject in (/home/www/glpi/vendor/sabre/vobject/lib)
laminas/laminas-i18n in (/home/www/glpi/vendor/laminas/laminas-i18n/src)
laminas/laminas-servicemanager in (/home/www/glpi/vendor/laminas/laminas-servicemanager/src)
monolog/monolog in (/home/www/glpi/vendor/monolog/monolog/src/Monolog)
sebastian/diff in (/home/www/glpi/vendor/sebastian/diff/src)
donatj/phpuseragentparser in (/home/www/glpi/vendor/donatj/phpuseragentparser/src/UserAgent)
elvanto/litemoji in (/home/www/glpi/vendor/elvanto/litemoji/src)
symfony/console in (/home/www/glpi/vendor/symfony/console)
scssphp/scssphp in (/home/www/glpi/vendor/scssphp/scssphp/src)
laminas/laminas-mail in (/home/www/glpi/vendor/laminas/laminas-mail/src/Protocol)
laminas/laminas-mime in (/home/www/glpi/vendor/laminas/laminas-mime/src)
rlanvin/php-rrule in (/home/www/glpi/vendor/rlanvin/php-rrule/src)
blueimp/jquery-file-upload in (/home/www/glpi/vendor/blueimp/jquery-file-upload/server/php)
ramsey/uuid in (/home/www/glpi/vendor/ramsey/uuid/src)
psr/log in (/home/www/glpi/vendor/psr/log/Psr/Log)
psr/simple-cache in (/home/www/glpi/vendor/psr/simple-cache/src)
psr/cache in (/home/www/glpi/vendor/psr/cache/src)
league/csv in (/home/www/glpi/vendor/league/csv/src)
mexitek/phpcolors in (/home/www/glpi/vendor/mexitek/phpcolors/src/Mexitek/PHPColors)
guzzlehttp/guzzle in (/home/www/glpi/vendor/guzzlehttp/guzzle/src)
guzzlehttp/psr7 in (/home/www/glpi/vendor/guzzlehttp/psr7/src)
glpi-project/inventory_format in (/home/www/glpi/vendor/glpi-project/inventory_format/lib/php)
wapmorgan/unified-archive in (/home/www/glpi/vendor/wapmorgan/unified-archive/src)
paragonie/sodium_compat in (/home/www/glpi/vendor/paragonie/sodium_compat/src)
symfony/cache in (/home/www/glpi/vendor/symfony/cache)
html2text/html2text in (/home/www/glpi/vendor/html2text/html2text/src)
symfony/dom-crawler in (/home/www/glpi/vendor/symfony/dom-crawler)
twig/twig in (/home/www/glpi/vendor/twig/twig/src)
twig/string-extra in (/home/www/glpi/vendor/twig/string-extra)
symfony/polyfill-ctype not found
symfony/polyfill-iconv not found
symfony/polyfill-mbstring not found
symfony/polyfill-php80 not found

</details><details><summary>SQL replicas</summary><pre> 
Not active

</details><details><summary>Notifications</summary><pre> 
Way of sending emails: SMTP+TLS (glpi@xxxx.xxxx.xx@xxxx.xxxx.xx)

</details><details><summary>Plugins list</summary><pre> 
	datainjection        Name: Data injection                 Version: 2.10.0-rc2 State: Enabled

</pre></details>

### Anything else?

I can't describe it in more detail

**Comment by cconard96** [comment]
Can you test this with the RC3 release that was just released earlier today? This issue should have been fixed by #10950.

**Comment by bielu22** [comment]
I have upgraded to RC3 and problem has been solved.
Thank you very much :)
